node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -1596,25 +1552,21 @@ mod tests {
                 .write_all(b"dns-servers = \"111.111.111.111,222.222.222.222\"\n")
                 .unwrap();
         }
-        let subject = NodeConfiguratorStandardPrivileged::new();
-
-        let configuration = subject
-            .configure(
-                &[
-                    "".to_string(),
-                    "--data-directory".to_string(),
-                    home_dir.to_str().unwrap().to_string(),
-                ],
-                &mut FakeStreamHolder::new().streams(),
-            )
-            .unwrap();
+        let directory_wrapper = make_pre_populated_mocked_directory_wrapper();
+
+        let (multi_config, _, _) = gathered_params_for_service_mode(
+            &directory_wrapper,
+            &convert_str_vec_slice_into_vec_of_strings(&[
+                "",
+                "--data-directory",
+                home_dir.to_str().unwrap(),
+            ]),
+        )
+        .unwrap();
 
         assert_eq!(
-            configuration.dns_servers,
-            vec![
-                SocketAddr::from_str("111.111.111.111:53").unwrap(),
-                SocketAddr::from_str("222.222.222.222:53").unwrap(),
-            ]
+            value_m!(multi_config, "dns-servers", String).unwrap(),
```

Comment:
The functionality leaving the bad feeling on you that stays untested now is a part of other tests so I won't take back here because .configure no longer has the ability of reading from the config file, it has been delegated to a different function, truly tested here. However .configure() still does what it did with the socketaddresses which has been test-covered more times.